### DISCUSSION TITLE
Altitude correction

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#218] [Q&A] [answered]
On tour again at the weekend->everything works fine!

Rather a problem of my 2 smartphones?
My smartphones both show too much about 60m for the altitude...
According to internet research, the problem is common...
Is it possible to do a manual correction?

**Comment by devemux86** [comment, marked as answer]
> On tour again at the weekend->everything works fine!

:+1: 

@Leeb64 if you mean the altitude above sea level, it is available in "Settings | Location | Advanced | Altitude | Sensor (MSL)".

Some devices may correct the GPS altitude automatically, so it is an option for users to check on their devices.

**Reply by Cer-o** [reply]
I understand that the altitude information is provided by the device and not by the map you are using?

**Reply by devemux86** [reply]
> I understand that the altitude information is provided by the device and not by the map you are using?

The altitude is provided by the GPS, it is usually above the WGS84 reference ellipsoid.
Some manufacturers may "correct" it to report the altitude above mean sea level (MSL).

Cruiser offers the altitude option to select the system's value or the extra MSL "correction".
You can also check the altitude with a specialized utility like the [GPSTest app](https://play.google.com/store/apps/details?id=com.android.gpstest).

**Reply by Cer-o** [reply]
Hello, I have installed the application and this is the information that it offers me when I am at home.

![image](https://github.com/devemux86/cruiser/assets/122309655/df5b57a8-21ad-4cba-8dfd-a6b62a1b6243)

And this is Cruiser with the two available Altitude settings.

![image](https://github.com/devemux86/cruiser/assets/122309655/75df41c2-4efe-44eb-a6f4-6fefb6a87e80)

![image](https://github.com/devemux86/cruiser/assets/122309655/233418a1-8d48-47aa-b2b4-54b74ccf72f3)

![image](https://github.com/devemux86/cruiser/assets/122309655/b3356ded-9a98-4bff-94d9-a7f5a7ce4702)

What is the correct configuration? .... by the way, what is the altitude at which I am at home?

**Reply by devemux86** [reply]
@Cer-o GPSTest app shows both altitudes:
- "Normal" value above an elliptical earth model
- "Corrected" MSL value above mean sea level

On your device to get the "corrected" altitude, it seems that you have to select the (MSL) Cruiser option.

**Comment by Leeb64** [comment]
OK! now its correct!
